README.md

```diff
@@ -79,21 +81,29 @@ For a default CMake build, the steps are the following:
 2. perform an out-of-source build:
 
         mkdir build && cd build
-        cmake ..
+        cmake .. (CMake options)
         make && make test && sudo make install
 
+### CMake options
 
+In general, please set and pass the following options to the `cmake ..` command:
 
-### CMake options
+* `-DCMAKE_INCLUDE_PATH=include_path`, with `include_path` the path to those includes that are not in default locations such as `/usr/local`.
+   For instance, the include path to the header files of libInt2, libCint, Eigen and the Intel MKL should be added to this variable.
 
-For this library, there are several extra options you can pass to the `cmake ..` command:
+* `-DCMAKE_LIBRARY_PATH=library_path`, with `library_path` the path to those libraries that are not in default locations such as `/usr/local`.
+   For instance, the libInt2, libCint and Intel MKL libraries should be on this search path.
 
-* `-DINSTALLATION_PREFIX=prefix`, with `prefix` (defaulted to `/usr/local`) the installation prefix you want the library to be installed it. This option controls where the library is installed:
+* `-DCMAKE_INSTALL_PREFIX=prefix`, with `prefix` (defaulted to `/usr/local`) the installation prefix you want the library to be installed it. This option controls where the library is installed:
     * the header files will be installed in `prefix/gqcp/include`
     * the compiled library will be installed in `prefix/gqcp/lib`
     * drivers (optional) and benchmarks (optional) will be installed in `prefix/gqcp/bin`
     * CMake target files will be installed in `prefix/gqcp/cmake`
 
+For this library, there are several extra options you can pass to the `cmake ..` command:
+
+* `-DEIGEN_USE_MKL_ALL=TRUE` makes sure that Eigen uses Intel MKL.
+
 
 * `-DBUILD_DOCS=ON` specifies that documentation should be built using Doxygen, in which case Graphviz is required for UML generation. A custom `docs` target will then be configured by CMake, so that
```

Comment:
```suggestion
* `-DBUILD_DOCS=TRUE` specifies that documentation should be built using Doxygen, in which case Graphviz is required for UML generation. A custom `docs` target will then be configured by CMake, so that
```